tests/models/test_models.py

```diff
@@ -1274,18 +1274,26 @@ def test_destvi(save_path):
     n_labels = 5
     n_layers = 2
     dataset = synthetic_iid(n_labels=n_labels)
+    dataset.obs["overclustering_vamp"] = list(range(dataset.n_obs))
     CondSCVI.setup_anndata(dataset, labels_key="labels")
     sc_model = CondSCVI(dataset, n_latent=n_latent, n_layers=n_layers)
     sc_model.train(1, train_size=1)
 
-    # step 2 learn destVI with multiple amortization scheme
+    # step 2 Check model setup
+    DestVI.setup_anndata(dataset, layer=None)
+    _ = DestVI.from_rna_model(dataset, sc_model, vamp_prior_p=10000)
```

Comment:
that seems extreme as a value for the test. Maybe unimportant.

Reply:
I added a comment. This vamp_prior_p is only used to check that preclustering in line 1277 contains less cells than vamp_prior_p (successful clustering outside of DestVI). Next line is then leading to ValueError. And overclustering is deleted afterwards.

Reply:
It has to be vamp_prior_p>dataset.n_obs to not have a value error, but I can set it to this value.

Reply:
Actually it has to be vamp_prior_p>n_largest_celltype. I am not using this more tight value but n_obs here. I don't think this test is critical but I imagine it's a functionality that could go wrong in future releases.